scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -545,6 +548,20 @@ static void update_stat_for_running(struct task_struct *p,
 	struct sys_stat *stat_cur = get_sys_stat_cur();
 	u64 wait_period, interval;
 
+	/*
+	 * If the sched_ext core directly dispatched a task, calculating the
+	 * task's deadline and time slice was also skipped. In this case, we
+	 * set the deadline and time slice here.
+	 *
+	 * Note that this is necessary when the kernel does not support
+	 * SCX_OPS_ENQ_MIGRATION_DISABLED or SCX_OPS_ENQ_MIGRATION_DISABLED
+	 * is not turned on.
+	 */
+	if (p->scx.slice == SCX_SLICE_DFL) {
```

Comment:
Nice! I'm wondering if we should have a per-task flag to determine in a more reliable way when the sched_ext core dispatches a task, so that we can update the slice, deadline, etc. accordingly.